src/services/export.service.js

```diff
@@ -21,11 +21,19 @@ const bulkExport = async (request, reply) => {
       })
     );
   }
+  if (parameters._bySubject) {
+    reply.code(400).send(
+      createOperationOutcome('The "_bySubject" parameter cannot be used in a system-level export request.', {
```

Comment:
I would say that we still want system level requests to allow for `_bySubject`. This would allow it to export every patient in the system for doing a calculation of everyone.

Reply:
Discussed and the Patient level export covers this functionality, so good to leave out System level